go.mod

```diff
@@ -53,7 +53,6 @@ require (
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
 	github.com/Masterminds/semver v1.3.1 // indirect
 	github.com/Masterminds/sprig v2.17.1+incompatible
-	github.com/Microsoft/go-winio v0.4.14 // indirect
```

Comment:
can you remove these go.* changes? the rest of the code isn't changing any imports so I'm not sure why these would need to be removed. They're also causing a git conflict so removing the changes is probably the easiest resolution.